includes/core/classes/class-assets.php

```diff
@@ -321,6 +321,7 @@ protected function localize( int $post_id ): array {
 				'eventRestApi'    => $event_rest_api,
 				'loginUrl'        => $this->get_login_url( $post_id ),
 				'registrationUrl' => $this->get_registration_url( $post_id ),
+				'siteUrl'         => get_home_url(),
```

Comment:
for consistency should it also be called `homeUrl`?

Reply:
Yes, makes sense. Sorry.